### ISSUE TITLE
XAML Controls Gallery : Open file/Save/Bold/Italic/Font Color buttons in RichEditBox high contrast improvements.

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/Xaml-Controls-Gallery#461]
Copied from internal [issue](https://microsoft.visualstudio.com/OS/_workitems/edit/26097138?src=WorkItemMention&src-action=artifact_link).

**Pre-requisite: ​**
Turn on High Contrast Black Mode. (Keyboard Shortcut: Alt+Shift+PrintScreen).

**Repro steps :**
Launch XAML Controls Gallery.
Activate RichEditBox  from All Controls Page.
Go to the richeditbox present under the header "A Custom Editor With RichEditBox".
Now observe hovering mouse over the buttons "Open file/Save/Bold/Italic/Font Color".

**Expected result:** 
High-Contrast Black/White Mode: On hovering the mouse over those buttons they should change color.

**Actual result:** 
High-Contrast Black Mode/White Mode: There is no change in color of the buttons.

![HC White Mode (1)](https://user-images.githubusercontent.com/7758786/83690938-78ea5a00-a5a6-11ea-9101-0203c839be1c.png)

**Comment by chingucoding** [comment]
@stmoy Should we remove the border for all themes or high contrast to fix this issue?

**Comment by YuliKl** [comment]
The lack of change feels wrong, and seems unique to this set of buttons. For example, the Back and Hamburger buttons, which are fairly similar in appearance, do get a colored backplate on hover. Ideally the buttons in the editor should have the same visual style: SystemColorHighlightColor background on hover.
